views/singleArticle/SingleArticle.jsx

```diff
@@ -1,23 +1,67 @@
 import { useParams } from "react-router-dom"
-import { getArticleById } from "../../api"
+import { getAllUsers, getArticleById, getArticleComments, updateArticleVote } from "../../api"
 import { useEffect, useState } from "react"
 
 import moment from 'moment'
 
-
 import styles from './SingleArticle.module.css'
 
+import Comment from "../../components/comment/Comment"
+
 function SingleArticle() {
   const { article_id } = useParams()
   const [ article, setArticle ] = useState({})
+  const [ viewComments, setViewComments ] = useState(false)
+  const [ articleComments, setArticleComments ] = useState([])
 
   useEffect(() => {
     getArticleById(article_id)
       .then((response) => {
         setArticle(response.article)
       })
+
+    getArticleComments(article_id)
```

Comment:
consider moving this logic and the rendering of the comments into their own comment list component to separate the logic from the articles logic